extensions/cornerstone-dicom-rt/src/getSopClassHandlerModule.ts

```diff
@@ -94,12 +95,19 @@ function _getDisplaySetsFromSeries(
     displaySet.referencedDisplaySetInstanceUID = referencedDisplaySet.displaySetInstanceUID;
   }
 
-  displaySet.load = ({ headers }) => _load(displaySet, servicesManager, extensionManager, headers);
+  displaySet.load = ({ headers, createSegmentation = true }) =>
+    _load(displaySet, servicesManager, extensionManager, headers, createSegmentation);
 
   return [displaySet];
 }
 
-function _load(rtDisplaySet, servicesManager: AppTypes.ServicesManager, extensionManager, headers) {
+function _load(
+  rtDisplaySet,
+  servicesManager: AppTypes.ServicesManager,
```

Comment:
Could you pass in an ExtensionParams instead of making changes when a different set of parameters for the basic parameters is required?  Needing to know the exact set of parameters is hard when there are so many functions that needs some sort of ExtensionParams like values.  If you want to define a sub-type of it that is fine, but it should be an object containing these, or one of these should be retrieved from the other one.